test/e2e/e2e_suite_test.go

```
@@ -48,8 +52,22 @@ var _ = BeforeSuite(func() {
 	isSNO, err = utilsHelpers.IsSNOCluster()
 	Expect(err).NotTo(HaveOccurred())
 
-	isBM, err = utilsHelpers.IsBMInfra()
-	Expect(err).NotTo(HaveOccurred())
+	isBM = false
+	platformType, _ = os.LookupEnv("PLATFORM_TYPE")
+	if platformType == string(generate.PlatformBaremetal) { // if it set
```

This check does not say the platform type is set, it says it's sell to **bm**.
You should check the error from the `os.LookupEnv("PLATFORM_TYPE")`, if there is an error it's not set.
Also, if `PLATFORM_TYPE` is set to an invalid type (not bm ar aws), the test should either fail or continue with a msg saying the var is not set and we are using a different value.
We need to decide what is out "default" reaction for when the var is not set.